src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_USERS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[NUMBER_OF_USERS];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+
+        for (Ball ball: balls) {
+            System.out.println(ball.toString());
+        }
```

Comment:
Your array is redundant, because you can print info about 3 random balls without array and you can achieve this with only one cycle. Also please pay attention, that `ball.toString()` contains redundant call of `.toString()` method, because under-the-hood of `System.out.println()` we have a call of `.toString()` method of `ball`.